### ISSUE TITLE
The Select component isn't keyboard accessible

### ISSUE THREAD

**Issue by a-trost** [antonreshetov/vue-form-components#31]
I've tried this a few times and I can't seem to figure out how to get the select component to be keyboard accessible. If the select component is selected and you hit enter it just toggles the first item in the list. This happens with both the regular and multi-select feature.

**Comment by BogdanCerovac** [comment]
@a-trost  - it looks like the whole project is missing (some) accessibility features and the select one is just the most obvious one. I was just scanning the code because it looks like a very good candidate for one of my projects but somebody would have to make some effort and make better use of "aria" tags etc. If I will have the time I can make some PR's but would need some help testing the accessibility improvements with screenreaders etc.
